docs/src/examples/components/List/Types/ListExampleActive.shorthand.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { List, Image } from '@stardust-ui/react'
+
+class ListExampleActive extends React.Component<any, any> {
+  state = {
+    key: 'irving',
+  }
```

Comment:
Let's try to save as many lines as possible in example files.  They need to fit vertically split on a laptop screen to be useful.

```suggestion
state = { key: 'irving' }
```